ci-operator/step-registry/smongiar/deploy-resources/smongiar-deploy-resources-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: smongiar-deploy-resources
+  from: test-image
+  commands: smongiar-deploy-resources-commands.sh
+  credentials:
+    - namespace: test-credentials
+      name: cluster-secrets
```

Comment:
@smongiar can you check if this `cluster-secrets` credentials exist in a vault?

Reply:
Thanks, you are right. I stored credentials in https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/foo-qe/cluster-secrets

Is there a way to reference those? perhaps, should I use `foo-qe` as namespace?
![image](https://github.com/openshift/release/assets/63113013/baf94cfe-2600-4e2b-8f00-df25a30562d8)

Reply:
@smongiar we also add two more key-value pair at this vault path, like given [here](https://cspi-qe.gitbook.io/ocp-ci-onboarding/tutorials/secrets_guide#create-a-new-secret) in the doc.
```
secretsync/target-namespace: "test-credentials"
secretsync/target-name: <name-of-secret> 
```

Reply:
sorry, could I ask where to put those, I can't find any valuable example on the github repo about that.
Do you mean something like:
`ref:
  as: smongiar-deploy-resources
  from: test-image
  commands: smongiar-deploy-resources-commands.sh
  credentials:
    - namespace: test-credentials
      name: cluster-secrets
      mount_path: /tmp/secrets/smongiar-qe
    - secretsync/target-namespace: "test-credentials"
      secretsync/target-name: "quay-io-auth-secret"
    - secretsync/target-namespace: "test-credentials"
      secretsync/target-name: "registry-redhat-io-auth-secret"
`

Reply:
This will be added in the vault same as attached in above ss, at https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/foo-qe/cluster-secrets